.env

```diff
@@ -1,8 +1,51 @@
+
+# General
+DEBUG_MODE=1
+NODE_ENV=development
+
+# Postgres
+POSTGRESQL_URL=postgresql://admin:admin@10.64.1.192:5432/${SERVICE_DB_NAME}
 POSTGRESQL_USER=admin
 POSTGRESQL_PASSWORD=admin
-JWT_SECRET=jwt-secret
-SERVICE_JWT_SECRET=service-jwt-secret
+SERVICE_DB_NAME=amplication-server
+
+# Nest
+CORS_ENABLE=1
+PORT=3000
+HOST=http://localhost:3000
+CLIENT_HOST=http://localhost:3001
+
+# GraphQL
```

Comment:
I see a large amount of changes to the root-level `.env` which isn't per se related to the Nx migration.

I would like to keep it at envfiles per app.